### ISSUE TITLE
VFS ValueError: invalid mode 'w'

### ISSUE THREAD

**Issue by lasso-codie** [TileDB-Inc/TileDB-Py#317]
When I run the following code:

```python
import tiledb
vfs = tiledb.VFS()
fh = vfs.open("./test/file", "w")
```
I get the following error:
`ValueError: invalid mode 'w'`

though the [docs](https://tiledb-inc-tiledb-py.readthedocs-hosted.com/en/stable/python-api.html#tiledb.VFS.open) clearly state that this is correct.

I'm using `tiledb=0.6.1` (`libtiledb=2.0.2`) on Windows with `Python 3.7.3 (default, Apr 24 2019, 15:29:51) [MSC v.1915 64 bit (AMD64)] :: Anaconda, Inc. on win32`

**Comment by Shelnutt2** [comment]
@lasso-codie looks like this is an oversight on our part. I'll let @ihnorton clarify if the docs are wrong or the code is wrong. For now you can fix this issue by using `wb` for the mode.

Here is the relevant line where we switch and expect `wb`.
https://github.com/TileDB-Inc/TileDB-Py/blob/32dc9314ed0f3f483147008457d1a24224d957de/tiledb/libtiledb.pyx#L5297

**Comment by lasso-codie** [comment]
Thanks for the fast response. Docs can outlive refactoring quickly, it works with this change. Seems there was an API change somewhere in the past.

**Comment by ihnorton** [comment]
This was a doc rendering issue, fixed by #315 and configuration changes. The current [stable docs](https://tiledb-inc-tiledb-py.readthedocs-hosted.com/en/stable/python-api.html#tiledb.VFS.open) (as linked above) show
![image](https://user-images.githubusercontent.com/327706/87743772-b05c4100-c7b7-11ea-8395-ec9c27931545.png)
Thank you!
